app/views/editor/rich.html.erb

```diff
@@ -518,4 +522,44 @@
       console.log("Draft:", editor.data.draft);
   }
 
+  var previewWindow = null;
+
+  function preview() {
+    let url = 'notes/preview'
+    $.ajax({
+      type: "POST",
+      url: url,
+      data:
+        {
+          'title': editor.titleModule.value(),
+          // 'image': editor.mainImageModule,
```

Comment:
So when we upload image, they are saved in backend. I think we can simply pass the url

Reply:
Okay when we upload an image, then a post call `/image` goes to backend.
The response of this api, we can url and `id`. We are interested in this `id` only, we can pass this `id` in the same manner as it get passed when we publish post (inside param: `main_image `)

Reply:
This is what I've come up with so far..

```suggestion
    let image_url = editor.mainImageModule.image.src
    let image_id = image_url.split("/").pop()
```